### DISCUSSION TITLE
Beginner question: name of default container?

### DISCUSSION THREAD

**Discussion by emendelson** [lima-vm/lima#985] [Q&A] [answered]
Apologies for a beginner questions, but I've spent hours searching for the answers without success. 

I've installed lima via Brew and figured out how to make my home folder writable by modifying lima.yaml and changing the mounts: section.

But now: how do I restart lima? I tried running `nerdctrl restart` - but this needs the name of the container I'm trying to restart and I can't see any way to figure out what the name of the container might be. I've tried `default` but that doesn't work. Is there any way to list the containers that might be running? Thank you for any help about things that ought to be obvious. 

(I restarted the machine in order to restart lima because I couldn't figure out any other way - but I'm obviously missing something obvious.

**Comment by AkihiroSuda** [comment, marked as answer]
Probably you wanted this

```bash
limactl stop default
limactl start default
```

Lima does not work inside a container (but containers work inside Lima), so `nerdctl restart` is irrelevant to restarting Lima.

**Reply by emendelson** [reply]
That is *exactly* what I was looking for. Thank you. (And I should have found it myself...)

**Comment by afbjorklund** [comment]
Maybe add `limactl restart` ?

**Reply by AkihiroSuda** [reply]
SGTM
